.eslintrc.js

```diff
@@ -26,15 +27,23 @@ module.exports = {
     react: {
       version: 'detect',
     },
+    'mdx/code-blocks': true,
   },
   overrides: [
     { files: ['src/**/*.jsx'] }, // eslint would lint .js only by default
     {
-      files: ['**/*.{md,mdx}'],
+      files: ['**/*.mdx'],
+      extends: ['plugin:mdx/recommended'],
+      rules: {
+        semi: ['off'],
+      },
+    },
+    {
+      files: ['**/*.md'],
```

Comment:
It can be linted by `eslint-plugin-mdx` as-is now, with `remark-lint` support.

Reply:
This repository use `markdownlint` at the moment so I'll leave it as it is.